pkg/server/chaosd/network.go

```diff
@@ -308,29 +308,49 @@ func (s *Server) applyEtcHosts(attack *core.NetworkCommand, uid string, env Envi
 		line = line + "\n"
 		_, err := w.WriteString(line)
 		if err != nil {
-			return errors.WithStack(err)
+			return perrors.WithStack(err)
 		}
 	}
 	// if not match any, then add a new line.
 	if newFlag {
 		_, err := w.WriteString(attack.DNSIp + "\t" + attack.DNSDomainName + "\n")
 		if err != nil {
-			return errors.WithStack(err)
+			return perrors.WithStack(err)
 		}
 	}
 
 	err = w.Flush()
 	if err != nil {
-		return errors.WithStack(err)
+		return perrors.WithStack(err)
 	}
 	err = fd.Sync()
 	if err != nil {
-		return errors.WithStack(err)
+		return perrors.WithStack(err)
 	}
 	recoverFlag = false
 	return nil
 }
 
+func (s *Server) applyFlood(attack *core.NetworkCommand) error {
+	cmd := bpm.DefaultProcessBuilder("bash", "-c", fmt.Sprintf("iperf -u -c %s -t %s -p %s -P %d -b %s", attack.IPAddress, attack.Duration, attack.Port, attack.Parallel, attack.Rate)).
```

Comment:
Why just UDP here?

Reply:
Because UDP provides unreliable transport, it can increase network traffic even without a server

Reply:
It's more easier to use, users can generate network traffic without deploying a server.